data/microos/butane/script

```diff
@@ -15,9 +15,7 @@
 #   8) set test hostname 
 #   9) test networking
 #
-# Redirect output to the console
-exec > >(exec tee -a /dev/console) 2>&1
-set -eux
+set -euox pipefail
```

Comment:
Not technically wrong, but the option is `-o pipefail` so it's IMO better to write it as

```suggestion
set -euxo pipefail
```